packages/nimble/R/nimbleFunction_core.R

```diff
@@ -117,6 +117,30 @@ nimbleFunction <- function(setup         = NULL,
     return(generatorFunction)
 }
 
```

Comment:
Oops that comment may have been residue of initial thoughts.  The Wiki page linked from the PR online has a full demo.  It is [here](https://github.com/nimble-dev/nimble/wiki/Developer-backdoor-to-manually-replace-generated-C).

The test failure seems to be occurring on multiple PRs.  I will look into it later, but I am fairly sure it is some last bump in all the test updating, not related to this particular PR or feature.